TodoApi/TodoApi/Controllers/TodosController.cs

```diff
@@ -83,6 +84,9 @@ public async Task<IActionResult> PutTodo(long id, Todo todo)
         [Authorize(Policy = "create")]
         public async Task<ActionResult<Todo>> PostTodo(Todo todo)
         {
+            DateTime dateCreated = DateTime.UtcNow.Date;
+            todo.DateCreated = dateCreated;
```

Comment:
> If I create something at 10pm CT, should its DateCreated be captured as "tomorrow" because it's past midnight UTC?
I think so. 

Sounds good! Just pushed a commit that removes Utc